packages/report-server/src/reportBuilder/context/types.ts

```diff
@@ -4,7 +4,8 @@
  */
 
 export interface Context {
-  orgUnits: { code: string; name: string }[];
+  orgUnits?: { code: string; name: string }[];
```

Comment:
Decided to make all these optional, rather than needing to maintain a default value for each one. It does mean that the actual context dependant functions now need to assert that a value is present for their dependencies, but I don't actually mind that as it provides a check to ensure that the developer has correctly wired up the dependencies 👍